resources/views/frontend/user-collection/show.blade.php

```diff
@@ -18,18 +18,19 @@
                 @if($items->isNotEmpty())
                     <p>{{ trans('user-collection.content-intro') }}</p>
                     <p>{{ trans('user-collection.content-usage') }}</p>
-                    @if (Experiment::is('WEBUMENIA-1654-shared-user-collections-with-scena'))
-                        <p>{{ trans('user-collection.share-info') }}</p>
-                    @endif
+                    <p>{{ trans('user-collection.share-info') }}</p>
                 @else
-                    <p class="alert alert-info text-center">{{ trans('user-collection.empty') }}</p>
+                    <div class="text-center text-muted">
+                        <p>{{ trans('user-collection.empty') }}</p>
+                        <img src="{{ asset('/images/no-image/diela/no-image-o.jpg') }}" class="w-50" />
+                        <p>{{ trans('user-collection.empty_hint') }} <i class="fa fa-star-o"></i>.</p>
```

Comment:
```suggestion
                        <p>{{ trans('user-collection.empty_hint') }}&nbsp;<i class="fa fa-star-o"></i>.</p>
```
(because of mobile)

<img width="307" alt="Screenshot 2021-11-19 at 14 00 01" src="https://user-images.githubusercontent.com/2682941/142626882-8219dcd4-e9cb-4a1b-b413-4d470ac41554.png">